snyk-monitor/values.yaml

```diff
@@ -94,6 +94,9 @@ no_proxy:
 use_keepalive: true
 skip_k8s_jobs:
 
+# Scan networking resources (Service, Ingress, Networking CRDs)
+scan_networking_resources: false
```

Comment:
Nit: You would need to do the same for `snyk-operator-certified/helm-charts/snyk-monitor/values.yaml` too 😊